DemoApp/Assets/MaxSdk.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Doesn't this file come with the `MaxSdk.cs` file?

Reply:
Looks like this is a meta file for the MaxSdk as a whole, there's a separate `MaxSdk.cs.meta` for the `MaxSdk.cs` file.